components/director/pkg/cert/certutils.go

```diff
@@ -220,3 +220,30 @@ func ParseCertificateBytes(cert []byte, key []byte) (*tls.Certificate, error) {
 
 	return &tlsCert, nil
 }
+
+func SubjectsMatch(actualSubject, expectedSubject string) bool {
+	return GetCommonName(expectedSubject) == GetCommonName(actualSubject) &&
+		GetCountry(expectedSubject) == GetCountry(actualSubject) &&
+		GetLocality(expectedSubject) == GetLocality(actualSubject) &&
+		GetOrganization(expectedSubject) == GetOrganization(actualSubject) &&
+		matchOrganizationalUnits(GetAllOrganizationalUnits(actualSubject), GetAllOrganizationalUnits(expectedSubject))
+}
+
+func matchOrganizationalUnits(actualOrgUnits, expectedOrgUnits []string) bool {
+	if len(expectedOrgUnits) != len(actualOrgUnits) {
+		return false
+	}
+
+	expectedOrgUnitsMap := make(map[string]struct{}, len(expectedOrgUnits))
+	for _, expectedOrgUnit := range expectedOrgUnits {
+		expectedOrgUnitsMap[strings.TrimSpace(expectedOrgUnit)] = struct{}{}
+	}
+
+	for _, actualOrgUnit := range actualOrgUnits {
+		if _, exist := expectedOrgUnitsMap[strings.TrimSpace(actualOrgUnit)]; !exist {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
Seems to be a duplicate of https://github.com/kyma-incubator/compass/blob/main/components/hydrator/internal/subject/cert_subject.go#L130-L155 ? Since it's in the director `pkg` and this package is imported in the hydrator component, we could use it there as well?

Reply:
It is already removed and adopted? Which commit are you looking at?

Reply:
My bad, I overlook the the commit with the changes.